numpy/distutils/fcompiler/gnu.py

```diff
@@ -160,9 +161,13 @@ def get_flags_linker_so(self):
         return opt
 
     def get_libgcc_dir(self):
-        status, output = exec_command(
-            self.compiler_f77 + ['-print-libgcc-file-name'], use_tee=0)
-        if not status:
+        try:
+            output = subprocess.check_output(self.compiler_f77 +
```

Comment:
There's a long comment in `exec_command` about passing `universal_newlines=False` here - are you sure it was incorrect?

Reply:
I think it is probably correct but `universal_newlines=False` is the default for `subprocess.check_output`, so perhaps less convincing that we still need a paragraph explaining why the default value is used

Reply:
Ah, the default is `False`, so this is fine